v3/lints/cabf_br/lint_subject_contains_organizational_unit_name.go

```diff
@@ -48,18 +49,14 @@ func NewSubjectContainsOrganizationalUnitName() lint.LintInterface {
 }
 
 func (l *SubjectContainsOrganizationalUnitName) CheckApplies(cert *x509.Certificate) bool {
-	return util.TypeInName(&cert.Subject, util.OrganizationalUnitNameOID)
+	return !cert.NotBefore.Before(time.Date(2022, time.September, 1, 0, 0, 0, 0, time.UTC))
```

Comment:
Perhaps @christopher-henderson is good with landing https://github.com/zmap/zlint/pull/644 ?

Reply:
A minor change to align with #644 has been made